src/data/fullness.txt

```diff
@@ -877,6 +877,7 @@ spooky stir-fry	2	3	good	4-7	0	0	14-18
 Spooky Surprise Egg	1	1	crappy	1	0	0	0	series 2 tiny plastic
 square CRIMBCOOKIE	2	1	awesome	8-10	0	0	0	10 Square to be Hip (+10% mox)
 squirmy violent party snack	1	1	good	2-4	2-20	2-20	2-20	10 Broken Dancing (Legendary Beat quest)
+St. Pete's sneaky smoothie	1	1	EPIC	5-7	0	0	20-40	30 Gritty Soul (+5 Moxie, +50% Moxie, +5 Spooky Resistance)
```

Comment:
here it's sorting capital S after lowercase s, but elsewhere you don't do that.

I would rather we say it sorts case sensitively because it makes it easier for me (I can just run `:sort` and be done with it), but that's not how it currently is in many places

Reply:
everywhere I am sorting text in this i am doing case insensitive sort. I think it makes more sense that way and it does seem to be precedent